android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -109,6 +109,13 @@ constructor(
       .read(SharedPreferenceKey.PRACTITIONER_ID.name, null)
       ?.extractLogicalIdUuid()
   }
+  private var familyLogicalIdActionParam: ActionParameter? = null
```

Comment:
I don't think we want to use `familyLogicalId` anywhere, that's an arbitrary name in a config, this applies to `familyLogicalId` used throughout the code. I'm not entirely certain, but I think the abstraction is that we want a list of `updateableOnEditIds` that collects all the values where the paramType is `UPDATEABLEIDS` (or whatever we name it) cc @ellykits